### PR TITLE
Cleaned up Vagrantfile

### PR DESCRIPTION
**Pull request by maus-**
Forgot a line, my bad!

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by javuto** [comment]
ok to test

File: Vagrantfile [hunk 1]
```diff
@@ -6,25 +6,26 @@ Vagrant.configure("2") do |config|
...
-   %w{ubuntu12  ubuntu12/precise64},
+   %w{ubuntu12  ubuntu/precise64},
    %w{freebsd10 chef/freebsd-10.0}
  ].each do |machine|
   (name, box) = machine
-  config.vm.define name do |box|
+  config.vm.define name do |build|
     # Private network for NFS
-    box.vm.network :private_network, ip: "192.168.56.101"
+    build.vm.box = box
```
Comment (theopolis) [inline Vagrantfile]: Are you sure about `box`?

**Comment by javuto** [comment]
ok to test

**Comment by javuto** [comment]
ok to test

**Comment by maus-** [comment]
Fixed/Tested sorry for being a bad grandma

Image of grandma

**Comment by javuto** [comment]
ok to test

**Pull request merged**